crates/typst-library/src/text/mod.rs

```diff
@@ -497,6 +497,9 @@ pub struct TextElem {
     /// line, so a higher hyphenation cost can result in awkward justification
     /// spacing.
     ///
+    /// Note: Hyphenation costs will only be applied when the
```

Comment:
I think we could fold this into the previous paragraph to keep one paragraph per cost feature. Let's also mention that justify implies linebreaks: optimized.

Reply:
Done.